src/manual/peer2peer/js/main.js

```diff
@@ -1105,10 +1112,10 @@ function printHandler_(message, color) {
   }
   $('messages').innerHTML += '<span style="color:' + color + ';">' + message +
                             '</span><br>';
-  console.log(message);
   if (color === 'red') {
     throw new Error(message);
   }
+  console.log(message);
```

Comment:
I moved it so that it does not print the message twice when errors occur. It will still be printed for non errors (i.e. color !== red)